epidose/device/util.sh

```diff
@@ -27,6 +27,16 @@ FILTER=/var/lib/epidose/client-filter.bin
 # Location of update_filter_d sleep process file
 export SLEEP_UPDATE_FILTER_PID=/var/run/epidose_update_filter_sleep
 
+# Location of local epidose repository
+LOCAL_REPO=/home/epidose/epidose
+
+# Url of remote epidose repository
+REMOTE_REPO=https://github.com/eellak/epidose.git
+
+# Set username and email for git commands
+USERNAME=stefanos1316
+EMAIL=stefanos1316@gmail.com
+
```

Comment:
Don't use your account here.  In any case you are not committing, so you can put e.g. epidose.device, right?

Reply:
Ok, I will just use the _epidose_ as a username and the epidose@device.com as the email.